index.js

```diff
@@ -2,60 +2,77 @@ var dir = require("node-dir");
 var path = require("path");
 var fs = require("fs");
 
-module.exports = function(folder, cb){
+module.exports = function(folder, cb) {
 
-	dir.files(folder, function(err, files) {
-		if(err){
-			cb(err);
+	var cwd = process.cwd();
+	process.chdir(folder);
```

Comment:
I don't like the idea of scripts changing my directory. I'm not sure which is the right practice. If I have two scripts running with & what is the effect?